sys/sys/thr.h

```diff
@@ -55,6 +55,7 @@ struct thr_param {
     long	*parent_tid;		/* parent accesses the new TID here. */
     int		flags;			/* thread flags. */
     struct rtprio	*rtp;		/* Real-time scheduling priority */
+    /* XXXAR: should ddc always be here? It is only used by CHERIABI. */
```

Comment:
For now I think the ifdef is the right solution.  On CHERI-128 we could use spares, but on CHERI-256 we'll have to change the structure because there isn't 32 bytes of spare space.